bundle/deployplan/plan.go

```diff
@@ -34,6 +37,32 @@ func NewPlan() *Plan {
 	}
 }
 
+// LoadPlanFromFile reads a plan from a JSON file.
+func LoadPlanFromFile(path string) (*Plan, error) {
+	data, err := os.ReadFile(path)
```

Comment:
Potential OOM if user specifies wrong file. Can use streaming decoder on the `io.Reader`, or wrap it with `bufio.NewReader` first to avoid unnecessary read calls.